File: Find-the-Highest-Altitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        int maxi = 0, sum = 0 ; 
       for(int i = 0;i<gain.Length;i++){
            sum += gain[i];
            maxi = Math.Max(maxi,sum);
       } 
       return maxi ; 
    }
}

File: Find-the-Highest-Altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
       int maxi = 0, sum = 0 ; 
       for(int i = 0;i<gain.size();i++){
            sum += gain[i];
            maxi = max(maxi,sum);
       } 
       return maxi ; 
    }
};